libs/protocol-ui/src/components/molecules/Select/Select.tsx

```diff
@@ -8,6 +8,14 @@ import FormLabel from '../../atoms/FormLabel';
 import HelperText from '../../atoms/HelperText';
 import ErrorMessage from '../../atoms/ErrorMessage';
 
+export function SelectContainer(props: any) {
+  return (
+     <components.SelectContainer
+        {...props}
+        innerProps={Object.assign({}, props.innerProps, { 'data-cy': 'daoSelect-testing' })}
```

Comment:
We'll want to make this passed in via props optionally instead of hardcoded -- this'll result in every Select having that data-id